scripts/convert_blipdiffusion_to_diffusers.py

```diff
@@ -0,0 +1,221 @@
+'''
+This scripts requires you to build lavis from source, since the pip version doesn't have blip diffusion
+Follow instruction https://github.com/salesforce/LAVIS/tree/main
+'''
```

Comment:
```suggestion
'''
This script requires you to build `LAVIS` from source, since the pip version doesn't have BLIP Diffusion. Follow instructions here: https://github.com/salesforce/LAVIS/tree/main.
'''
```